cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/join/JoinDefinition.java

```diff
@@ -121,11 +129,25 @@ public Builder setOutputSchemaName(@Nullable String name) {
       return this;
     }
 
+    /**
+     * Set the distribution factor and stage name of the skewed stage. This should be set if the join being performed
+     * is skewed (ie. joining a large dataset with a small dataset) and the "small" dataset is too large to broadcast.
```

Comment:
can you add a line about how the given stage must be a required stage, and no stage should be broadcast.

Reply:
done